frontend/drivers/platform_wiiu.c

```diff
@@ -599,19 +604,19 @@ static ssize_t wiiu_log_write(struct _reent *r, void *fd, const char *ptr, size_
 
    wiiu_log_lock = 1;
 
-   int ret;
+   int sent;
    int remaining = len;
 
    while(remaining > 0)
    {
-      int block = remaining < 1472 ? remaining : 1472;
-      ret = sendto(wiiu_log_socket, ptr, block, 0, (struct sockaddr *)&broadcast, sizeof(broadcast));
+      int block = remaining < DGRAM_SIZE ? remaining : DGRAM_SIZE;
```

Comment:
..
you don't need to go overboard..
just keep it the way it was. no need to change something that already works... 576 is too low.

Reply:
No, it’s not. 

I’m trying to use numbers based on standards, either de facto (typical Ethernet MTU) or RFC. I really don’t understand why you’re so married to some arbitrary size you picked, other than the fact that you picked it.

576 is certainly not too low. The above method gets invoked indirectly via printf and friends, which are typically line-buffered. There simply isn’t any place in the code that is sending anywhere close to 500+ bytes on  a single line, and this is only ever going over a LAN so it doesn’t impact net play or other network functions.

Reply:
I didn't pick it, the original writer picked it, and it was a good choice for 99% of the cases.
but w/e, no need to discuss this further.